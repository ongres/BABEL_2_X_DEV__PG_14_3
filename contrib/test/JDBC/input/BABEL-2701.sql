create table babel_2701(a int)
GO

select object_name(object_id) from sys.objects where name = 'babel_2701';
GO

drop table babel_2701
GO