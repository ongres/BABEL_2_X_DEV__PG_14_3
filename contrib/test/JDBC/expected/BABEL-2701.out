create table babel_2701(a int)
GO

select object_name(object_id) from sys.objects where name = 'babel_2701';
GO
~~START~~
varchar
babel_2701
~~END~~


drop table babel_2701
GO
